### ISSUE TITLE
warn about soh and log in water current section of fdsnIdentifiers

### ISSUE THREAD

**Issue by crotwell** [iris-edu/mseed3-evaluation#23]
I thought I had posted this before, but can't find it, so sorry if a repeat.

In the Water Current section of the FDSNIdentifiers document, add

> Do not use SOH or LOG as these conflict with widely used state of health and logging channel identifiers.

**Comment by crotwell** [comment]
Right, but because this is a special exception to this specific instrument, and the only case in which an instrument does not determine its acceptable band/gain codes, I feel it would be best to add it directly to the water current section.

**Comment by chad-iris** [comment]
Got it, add and additional note about the special exceptions of LOG and SOH to the "O" instrument where they would potentially collide. 👍
